grafana-plugin/src/models/label/label.ts

```diff
@@ -3,34 +3,34 @@ import { action, observable } from 'mobx';
 import BaseStore from 'models/base_store';
 import { makeRequest } from 'network';
 import { RootStore } from 'state';
-import { openNotification } from 'utils';
+import { WithGlobalNotification } from 'utils/decorators';
 
 import { LabelKey, LabelValue } from './label.types';
 
 export class LabelStore extends BaseStore {
   @observable.shallow
-  public keys: LabelKey[] = [];
+  keys: LabelKey[] = [];
 
   @observable.shallow
-  public values: { [key: string]: LabelValue[] } = {};
+  values: { [key: string]: LabelValue[] } = {};
 
   constructor(rootStore: RootStore) {
     super(rootStore);
 
     this.path = '/labels/';
   }
 
-  @action
-  public async loadKeys() {
+  @action.bound
```

Comment:
what's the difference between @action.bound and changing the method to be an arrow function instead where we no longer miss the this context?

Reply:
Nice question, I encountered the same hesitation during development and here's what I've learnt :D 

The reason is that with arrow fn syntax, we switch from method to property decorator:
![image](https://github.com/grafana/oncall/assets/12073649/c40d21fa-d8e9-4037-95cf-9cad17078de5)

In property decorators [you don't have access](https://www.typescriptlang.org/docs/handbook/decorators.html#property-decorators) to the third argument of our decorator (descriptor).

In method decorator [you do have](https://www.typescriptlang.org/docs/handbook/decorators.html#method-decorators).

And having in mind that we will switch soon to `makeAutoObservable` when those `action.bound` decorators will simply disappear, it's more handy to have access descriptor when using decorators.